plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageQueryManager.java

```diff
@@ -168,19 +160,14 @@ public boolean performOk()
         if (checkQueryTypeDDL.getSelection()) queryTypes.add(DBCExecutionPurpose.META_DDL.name());
 
         Integer historyDays = UIUtils.getTextInteger(textHistoryDays);
-        Integer entriesPerPage = UIUtils.getTextInteger(textEntriesPerPage);
 
         DBPPreferenceStore store = DBWorkbench.getPlatform().getPreferenceStore();
         store.setValue(QMConstants.PROP_OBJECT_TYPES, QMObjectType.toString(objectTypes));
         store.setValue(QMConstants.PROP_QUERY_TYPES, CommonUtils.makeString(queryTypes, ','));
         if (historyDays != null) {
             store.setValue(QMConstants.PROP_HISTORY_DAYS, Math.max(1, historyDays));
         }
-        if (entriesPerPage != null) {
-            store.setValue(QMConstants.PROP_ENTRIES_PER_PAGE, Math.max(1, entriesPerPage));
-        }
         store.setValue(QMConstants.PROP_STORE_LOG_FILE, checkStoreLog.getSelection());
-        store.setValue(QMConstants.PROP_LOG_DIRECTORY, textOutputFolder.getText());
```

Comment:
But why do you want to remove QMConstants.PROP_LOG_DIRECTORY parameter? I do not understand.

Reply:
There is nothing to understand. I did that mistakenly.